x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -56,8 +56,8 @@ export interface RegistryPackage {
   type: string;
   categories: string[];
   requirement: RequirementsByServiceName;
-  screenshots?: ScreenshotItem[];
-  icons?: string[];
```

Comment:
Copying the definition from the registry https://github.com/elastic/package-registry/blob/master/util/package.go#L42-L43